test-requirements.txt

```
@@ -14,5 +14,5 @@ pytest-cov>=2.10.0,<3.0.0
 py>=1.5.2
 typed_ast>=1.4.0,<2; python_version>='3.8'
 virtualenv>=20.6.0
-setuptools!=50
+setuptools==61.0.0
```

```suggestion
setuptools!=50,<61.1.0
```
61.0.0 doesn't support 3.6

---

Yeah, I realized. I want to see if the Windows builds will pass. I'd rather pin to a version that supports all versions of python we support if possible, than use different versions of setuptools on different pythons, though.